src/get-started/install/_ios-setup.md

```diff
@@ -6,178 +6,316 @@ To develop Flutter apps for iOS, you need a Mac with Xcode installed.
 
  1. Install the latest stable version of Xcode
     (using [web download][] or the [Mac App Store][]).
- 1. Configure the Xcode command-line tools to use the
-    newly-installed version of Xcode by
-    running the following from the command line:
+
+ 1. To configure the Xcode command-line tools to use the
+    installed version, run the following commands.
 
     ```terminal
-    $ sudo xcode-select --switch /Applications/Xcode.app/Contents/Developer
+    $ sudo xcode-select -s /Applications/Xcode.app/Contents/Developer
```

Comment:
Yup, that's shorter, but these commands are equivalent.

Reply:
I prefer the long version for docs as they are more instructive. Personal preference, no issues either way.